libraries/AP_Camera/AP_Camera_Backend.cpp

```diff
@@ -105,6 +120,14 @@ bool AP_Camera_Backend::take_picture()
     return false;
 }
 
+// take multiple pictures, time_interval between two consecutive pictures is in miliseconds
+// total_num is number of pictures to be taken, -1 means capture forever
+bool AP_Camera_Backend::take_multiple_pictures(uint32_t time_interval, int16_t total_num)
+{
+    time_interval_settings = {time_interval, total_num};
+    return true;
```

Comment:
.. I'm not sure there's much point in returning a bool if it will always be true..